src/OrchardCore.Modules/OrchardCore.Media/Services/MediaCacheBackgroundTask.cs

```diff
@@ -0,0 +1,160 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Logging;
+using OrchardCore.BackgroundTasks;
+using OrchardCore.Environment.Shell;
+using OrchardCore.Media.Core;
+
+namespace OrchardCore.Media.Services
+{
+
+    // At 12:00 on Monday.
+    [BackgroundTask(Schedule = "0 12 * * 1", Description = "Performs cleanup operations for ms-cache and is-cache folders periodically.")]
```

Comment:
It would have been more useful to define task info using options the user can control using a UI. But this is outside the scope of this PR.

Reply:
The UI is enabled when you enable the Background Task feature in the admin. Then you can change the Schedule of the BG task there.

Reply:
Yeah cool. More improvement on that UI in this PR  #13046